### DISCUSSION TITLE
Spanning of one `desktop instance` across multiple screens.

### DISCUSSION THREAD

**Discussion by BlaiZephyr** [YaLTeR/niri#803] [Ideas]
first off, sorry if this is a duplicate or this is possible already - but i didn't find anything regarding this.

I've been wanting to run niri as a daily driver, but I'm missing a way to Span the view across multiple screens to have
 my 2 monitors to act as one big monitor, this way i could fully utilize the "scrolling" part of niri without having to deal with multiple screens and independent Work spaces for each monitor. This would allow for an easier way to create a multi desktop workspace for my tasks which i could switch to and from more easily.

Especially with horizontal Work spaces, I could have Firefox and discord on one, and vs code + terminals on the other for example.

Simply shifting your views to the sides but still having them in view would work well with multiple monitors.

**Comment by YaLTeR** [comment]
Hey, joining multiple physical monitors into one logical monitor isn't implemented and unfortunately would be quite a bit of work. Workspaces that are global across all monitors, rather than local to each monitor, would also be quite a bit of work because that's not how any of the niri code currently works. So I'm afraid this is not something that can be easily added.

**Reply by BlaiZephyr** [reply]
yeah i figured as much.. maybe a third party tool can help, i'll look into it. thanks for the quick answer!
